src/lib/ngsi/ContextAttribute.cpp

```diff
@@ -357,18 +357,28 @@ std::string ContextAttribute::toJson(bool isLastElement)
 {
   std::string  out;
   bool         isNumber = false;
+  bool         isString = true;
 
   if (type == "number")
   {
     isNumber = true;
   }
 
-  if (((type == "") || (isNumber == true)) && (metadataVector.size() == 0))
+  if(type == "string")
```

Comment:
If you think that part is fixable, I'd recommend to hadd a comment over these lines:

```
//FIXME P10: rendering code could be simplified a lot. To discuss with Ken, once he is back
```

or something like that. Otherwise, we are in risk of forgetting thinks (it had happended in the past ;)